include/moveit_grasps/grasp_filter.h

```diff
@@ -220,6 +220,14 @@ class GraspFilter
                             GraspCandidatePtr& grasp_candidate,
                             const moveit::core::GroupStateValidityCallbackFn& constraint_fn);
 
+  /**
+   * \brief  For suction grippers, check that at least one voxel has an overlap > some cutoff.
+   *         This is used as a pre-filter step before IK.
+   * \param grasp_candidates - All grasp candidates
+   * \param cutoff - some fractional cutoff where at least one voxel must have > cutoff overlap with the target
```

Comment:
I think something like `threshold` or `min_overlap` would be a bit more descriptive.